src/components/Navbar.js

```diff
@@ -7,7 +7,7 @@ function Nav() {
       <h1>BOOKSTORE CMS</h1>
       <ul>
         <li>
-          <Link to="/">BOOKS</Link>
+          <Link to="/form">BOOKS</Link>
```

Comment:
- [ ] **[OPTIONAL]** I think you should maintain the root path instead of `/form`